### PR TITLE
added print color option on list

### PR DESCRIPTION
**Pull request by Akhill2020**
**Description:** Added print calendar button list view also. also maqke some changes in js becuase it was added display block to script tage when closing print screen and it added some content on bottom so I have added condition in it.
**Clickup:** https://app.clickup.com/t/86cwk3mex
**After:** https://www.screenpresso.com/=hPEJ8yLicPhk

### CODE REVIEW

File: assets/js/default-calendar.js [hunk 1]
```diff
@@ -405,7 +405,7 @@ jQuery(function ($) {
 		window.print();
 
 		$('body > .simcal-calendar').remove();
-		$('body').children().show();
+		$('body').children().not('script').show();
```
Comment (rosinghal) [inline assets/js/default-calendar.js:408]: What is this for?

Comment (Akhill2020) [reply assets/js/default-calendar.js:408]: in js display block it was added display block to script tag when closing print screen, and it added some content on bottom, so I have added a condition in it. to skip the script tag.

**Review by rosinghal — APPROVED** [review]

File: assets/css/default-calendar-grid.css [hunk 2]
```diff
@@ -559,7 +559,12 @@
 	.simcal-print-calendar .print-calendar-button,
 	.print-calendar-button,
 	.simcal-print-calendar .simcal-nav .simcal-prev,
-	.simcal-print-calendar .simcal-nav .simcal-next {
+	.simcal-print-calendar .simcal-nav .simcal-next,
```
Comment (rosinghal) [inline assets/css/default-calendar-grid.css:562]: Why are we changing CSS of grid when we want this change for fullcalendar?

Comment (Akhill2020) [reply assets/css/default-calendar-grid.css:562]: this css is apply on the print screen and we have common css for grid list and fullcalendar grid view.

**Review by rosinghal — APPROVED** [review]

**Pull request merged**